tuf/api/metadata.py

```diff
@@ -291,22 +289,24 @@ def __init__(
     def from_dict(cls, signed_dict: JsonDict) -> 'Signed':
         """Creates Signed object from its JSON/dict representation. """
 
+        # Create empty object with default or parametrized constructor with
+        # default arguments.
+        obj = cls()
+        obj._type = signed_dict['_type']
+        obj.version = signed_dict['version']
+        obj.spec_version = signed_dict['spec_version']
         # Convert 'expires' TUF metadata string to a datetime object, which is
         # what the constructor expects and what we store. The inverse operation
         # is implemented in 'to_dict'.
-        signed_dict['expires'] = tuf.formats.expiry_string_to_datetime(
+        obj.expires = tuf.formats.expiry_string_to_datetime(
                 signed_dict['expires'])
-        # NOTE: We write the converted 'expires' back into 'signed_dict' above
-        # so that we can pass it to the constructor as  '**signed_dict' below,
-        # along with other fields that belong to Signed subclasses.
```

Comment:
Should the removal of this comment indicate we should no longer be doing the conversion of the expires string to a datetime object here?